src/components/about-us/AboutHero.tsx

```diff
@@ -1,13 +1,62 @@
-import { YouTubeEmbed } from "@/components/youtube-embed/YoutubeEmbed";
 import { FC } from "react";
+import Image from "next/image";
+import { Callout } from "@radix-ui/themes";
 
 const AboutHero: FC = () => (
-  <div className="py-8 md:py-20 mx-auto" style={{ maxWidth: 800 }}>
-    <YouTubeEmbed
-      title="About Distribute Aid!"
-      url="https://www.youtube.com/embed/msizPweg3kE"
-    />
-  </div>
+  <>
+    <div className="absolute inset-0 bg-navy-900 -z-10 h-[609px]"></div>
+    <Callout.Root color="green" className="bg-[var(--green-9)] rounded-none">
+      <Callout.Text>Some message here</Callout.Text>
+    </Callout.Root>
+    <div className=" h-[943px] mt-12 flex justify-center items-center flex-col gap-8 sm:gap-[40px] md:gap-[86px]">
+      <div className="text-[60px] tracking-[-0.16px] font-extrabold leading-[60px] text-center text-white">
+        Distribute Aid
+      </div>
+      <div className="text-[28px] md:text-[36px] leading-[40px] text-center px-3 text-white max-h-full">
```

Comment:
suggestion (non-blocking): Use RadixUI's [Text Component](https://www.radix-ui.com/themes/docs/components/text) if possible.  You can still include Tailwind classNames for properties that it doesn't handle.